README.md

```diff
@@ -1,21 +1,37 @@
-# AWS Mobile SDK for iOS
+# AWS SDK for iOS
 
-[![Release](https://img.shields.io/github/release/aws/aws-sdk-ios.svg)]()
-[![CocoaPods](https://img.shields.io/cocoapods/v/AWSiOSSDKv2.svg)]()
+[![Release](https://img.shields.io/github/release/aws/aws-sdk-ios.svg)](../../releases)
+[![CocoaPods](https://img.shields.io/cocoapods/v/AWSiOSSDKv2.svg)](https://cocoapods.org/pods/AWSiOSSDKv2)
+[![CocoaPods Downloads](https://img.shields.io/cocoapods/dt/AWSCore.svg)](https://cocoapods.org/pods/AWSiOSSDKv2)
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
-[![Twitter Follow](https://img.shields.io/twitter/follow/AWSforMobile.svg?style=social&label=Follow)](https://twitter.com/AWSforMobile)
 
-## Setting Up
+The AWS SDK for iOS provides a library and documentation for developers to build connected mobile applications using AWS.
 
-To get started with the AWS SDK for iOS, check out the [AWS Mobile Developer Guide for iOS](https://docs.aws.amazon.com/aws-mobile/latest/developerguide/getting-started.html). You can set up the SDK and start building a new project, or you integrate the SDK in an existing project. You can also run the samples to get a sense of how the SDK works.
+### Features / APIs
+
+- [__Authentication__](https://aws-amplify.github.io/docs/ios/authentication): APIs and building blocks for developers who want to create user authentication experiences.  
+- [__Analytics__](https://aws-amplify.github.io/docs/ios/analytics): Easily collect analytics data for your app. Analytics data includes user sessions and other custom events that you want to track in your app.  
+- [__API__](https://aws-amplify.github.io/docs/ios/api): Provides a simple solution when making HTTP requests. It provides an automatic, lightweight signing process which complies with AWS Signature Version 4.
+- [__Storage__](https://aws-amplify.github.io/docs/ios/storage): Provides a simple mechanism for managing user content for your app in public, protected or private storage buckets.  
+- [__Push Notifications__](https://aws-amplify.github.io/docs/ios/push-notifications): Allows you to integrate push notifications in your app with Amazon Pinpoint targeting and campaign management support.
+- [__PubSub__](https://aws-amplify.github.io/docs/ios/pubsub): Provides connectivity with cloud-based message-oriented middleware.  
+- [__Generated AWS Service Interfaces__](https://aws-amplify.github.io/docs/ios/manualsetup): Provides direct interaction with any AWS service. 
+
+#### Visit our [Web Site](https://aws-amplify.github.io) to learn more about AWS Amplify.
+
+* [Documentation](https://aws-amplify.github.io/docs)
+* [Setup](#setup)
+* [Examples](https://github.com/awslabs/aws-sdk-android-samples)
```

Comment:
This should point to the iOS version: `https://github.com/awslabs/aws-sdk-ios-samples`

Reply:
Good catch! I fixed it.